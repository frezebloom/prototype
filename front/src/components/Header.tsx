import * as React from 'react'
import { Link } from "react-router-dom"

import "../styles/header.scss"; 

interface HeaderProps {
  title: string
}

const Header: React.FC<HeaderProps> = ({ title }) => {
  return(
   
    <div className="header">
       <h1>{title}</h1>
    
          <Link to="/">Задачи</Link>
    

          <Link to="/token">Сшивка</Link>
  
    </div>
     
 
  )
}

export default Header