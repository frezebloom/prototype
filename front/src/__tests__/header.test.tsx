import * as React from 'react';
import { shallow } from 'enzyme';
import Header from '../components/Header';

describe('Header', () => {
  it('Matches the snapshot', () => {
    const wrapper = shallow(<Header title="MockTitle" />);
    expect(wrapper).toMatchSnapshot();
  })
});

