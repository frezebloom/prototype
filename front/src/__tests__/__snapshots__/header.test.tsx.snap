// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Matches the snapshot 1`] = `
<div
  className="header"
>
  <h1>
    MockTitle
  </h1>
  <Link
    to="/"
  >
    Главная
  </Link>
  <Link
    to="/token"
  >
    Token
  </Link>
</div>
`;
